src/axbolt/delayedmessage/delayedmessage.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+import "google/protobuf/any.proto"; 
+
+package axbolt.outbox;
+option go_package = "delayedmessage";
+
+// DelayedMessage is a container for holding delayed message-specific data that is
+// serialized into the protobuf format before being persisted in a Bolt bucket
+message DelayedMessage{
+    string id = 1;
+    string causation_id = 2;
+    string correlation_id = 3;
+    string created_at = 4;
+    string send_at = 5;
+    google.protobuf.Any message = 6;
+    int32 operation = 7;
+    string destination_endpoint = 8;
+} 
```

Comment:
There is now an `ax.EnvelopeProto` struct that could be used here.  It would cover all of the fields other than the operation and destination endpoint. You don't *have* to use it, but I think it makes sense.